client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableInterfaces.ts

```diff
@@ -15,7 +15,7 @@ import { IndexedInvestigationData } from './InvestigationTablesHeaders';
 
 export type StatusFilter = InvestigationMainStatusCodes[];
 export type SubStatusFilter = string[];
-export type DeskFilter = number[];
+export type DeskFilter = number[] | (number | null)[];
```

Comment:
If it can be number and null isn't it just ( number | null)[ ]?

Reply:
Not really, because in user managment and in adming landing paged the list dont have unassigned desks on , so null isnt an option there - it saves lots of unnecssary casting